src/System.Runtime.Extensions/tests/System/Reflection/AssemblyNameProxyTests.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Globalization;
+using Xunit;
+
+namespace System.Reflection.Tests
+{
+    public static class AssemblyNameTests
+    {
+       
+        [Fact]
+        public static void GetAssemblyName_AssemblyNameProxy()
+        {
+            AssemblyNameProxy anp = new AssemblyNameProxy();
+            Assert.Throws<ArgumentNullException>("assemblyFile", () => anp.GetAssemblyName(null));
+            Assert.Throws<ArgumentException>(() => anp.GetAssemblyName(string.Empty));
+            Assert.Throws<System.IO.FileNotFoundException>(() => anp.GetAssemblyName("IDontExist"));
```

Comment:
Nit: for good measure, maybe use something we can be much more confident won't exist, e.g. Guid.NewGuid().ToString("N")